bind-mount.c

```diff
@@ -399,14 +403,34 @@ bind_mount (int           proc_fd,
   if (resolved_dest == NULL)
     return 2;
 
-  mount_tab = parse_mountinfo (proc_fd, resolved_dest);
+  dest_fd = open (resolved_dest, O_PATH);
```

Comment:
Maybe `O_PATH | O_CLOEXEC` for consistency?

We close this fd before we have any opportunity to exec subprocesses *anyway*, but it's good to get into the habit of always using `O_CLOEXEC`.

Reply:
Right, fixed.